### PR TITLE
Port to Qt6

### PR DESCRIPTION
**Pull request by stefonarch**
Compiled without `lxqt2-` prefix.
screen_area_sab_18:29:20_

### CODE REVIEW

**Comment by stefonarch** [comment]
Recheck what happens [here](https://github.com/lxqt/lxqt-about/blob/e3453de23d1bd4e50a1a238952cc564feb1b8f98/aboutdialog/technicalinfo.cpp#L179)

```
    table->add(QStringLiteral("Translations"),         Translator::translationSearchPaths().join(QLatin1Char('\n')));
```

It generates this line (see also the first screenshot) where "openbox-qt6" is the name of the `.desktop` file in `/usr/share/xsessions/`

screen_area_lun_08:14:09_

File: translatorsinfo/translatorsinfo.cpp [hunk 1]
```diff
@@ -353,11 +354,11 @@ TranslatorsInfo::TranslatorsInfo()
 {
     //fillLangguages(&mLanguagesList);
 
-    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::IniFormat);
-    src.setIniCodec("UTF-8");
+    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::Format::IniFormat);
+    //Format("UTF-8");not working yet
```
Comment (stefonarch) [inline translatorsinfo/translatorsinfo.cpp:358]: Maybe not even needed?

**Comment by stefonarch** [comment]
Compiled on arch the weird translationdir is gone:

screen_area_mer_15:31:52_

File: translatorsinfo/translatorsinfo.cpp (hunk 1 above)
Comment (tsujan) [reply translatorsinfo/translatorsinfo.cpp:357]: Please keep the first line exactly as it was (it doesn't need any change) :

```c++
    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::IniFormat);
```

And just remove `src.setIniCodec("UTF-8");` below it — in Qt6, everything is in UTF-8.

File: translatorsinfo/translatorsinfo.cpp [hunk 2]
```diff
@@ -409,7 +409,8 @@ QString TranslatorPerson::asHtml() const
 
     if (!mContact.isEmpty())
     {
-        if (mContact.contains(QRegExp(QStringLiteral("^(https?|mailto):"))))
+         static QRegularExpression regexp(QStringLiteral("^(https?|mailto):"));
+    if (regexp.match(mContact).hasMatch())// to recheck if correct
```
Comment (tsujan) [inline translatorsinfo/translatorsinfo.cpp:413]: It should be like this:

```c++
        static const QRegularExpression regexp(QStringLiteral("^(https?|mailto):"));
        if (mContact.contains(regexp))
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -27,9 +27,10 @@ set(UI_FILES
 )
 
 # Minimum versions
-set(LXQT_MINIMUM_VERSION "1.4.0")
+set(LXQT_MINIMUM_VERSION "2.0.0")
 
 find_package(lxqt ${LXQT_MINIMUM_VERSION} REQUIRED)
+find_package(Qt6 COMPONENTS REQUIRED Core Widgets)
```
Comment (tsujan) [inline CMakeLists.txt:33]: Like before, this isn't needed.

**Comment by stefonarch** [comment]
Done, please recheck.

**Review by tsujan — APPROVED** [review]
GTM

**Pull request merged**